packages/corvid-local-site/src/codeCompletion.js

```diff
@@ -18,13 +19,21 @@ let corvidTypes,
 try {
   corvidTypes = require("corvid-types");
 
+  const SHARED_COMPILER_OPTIONS = {
+    composite: true,
+    noEmit: false,
+    skipLibCheck: true
+  };
+
   pageTsConfigContent = prettyStringify({
     extends: corvidTypes.configPaths.page,
     compilerOptions: {
       baseUrl: ".",
       paths: {
-        "public/*": ["../../public/*"]
-      }
+        "public/*": ["../../public/*"],
+        "backend/*": ["../../backend/*"]
```

Comment:
@agankarin 
This allows importing from backend (which is good), but unfortunately, typescript ignores .jsw files. Regarding our previous discussions on supporting backend, and corvid-magic-sauce of wrapping calls to webMethods, I realized we actually don't need to do anything special! 🕺 
The reason is, that the user should anyways write backend functions asynchronously. If they don't, they are making an error. So if the user writes async code in backend, and we will allow him to import functions from .jsw files, the secret sauce is all set and the user will get an awesome experience. The only minor (😆 ) problem left is that typescript does not seem to support  custom file extensions, and it doesn't sound like it will support them in the near future:
https://github.com/microsoft/TypeScript/issues/10939